### DISCUSSION TITLE
Import CSV with Custom Fields

### DISCUSSION THREAD

**Discussion by lucavimercatiovsr** [snipe/snipe-it#15657] [Q&A]
Hi All,
I'm trying to import about 80 computer asset while I set an additional custom field called "Hostname".
In the import windows I see the value assigned to the custom filed and assigned to the field but, when I confirm the import a new asset is created but in the asset there is no "Hostname" filed at all.

Here the import window:
![asset_hostname_1](https://github.com/user-attachments/assets/26a55646-c705-4c94-bc6d-6501263809d0)

Thank you!

**Comment by snipe** [comment]
You have to make sure the custom fieldset is associated with the model. 

https://snipe-it.readme.io/docs/custom-fields
https://snipe-it.readme.io/docs/importing#importing-custom-fields

**Comment by lucavimercatiovsr** [comment]
Thank you! I imported all asset with many models and assign the filedset to all model, then the field became visible.

Another Question:
Is there any way to reset the ID assigned to users? If i remove all user, also from soft deleted, the ID of addedd user continue from the last added +1, if I delete everything except for the admin (ID 1) and import again from csv  (I tried to add a column ID specifying to start from ID 2) I obtain the same result as attached screenshot.

![image](https://github.com/user-attachments/assets/cc47466f-1ed2-4638-b50d-36a700bdff93)

Thank you again!
